src/drawing/DrawingPathGroup.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { DrawingSVGRef } from './DrawingSVG';
+import { isVectorEffectSupported } from './drawingUtil';
+import { PathGroup, Position } from '../@types';
+import './DrawingPathGroup.scss';
+
+export type Props = {
+    isActive?: boolean;
+    pathGroup: PathGroup;
+    rootEl: DrawingSVGRef | null;
+};
+
+export const DRAWING_BORDER_WIDTH = 1;
+
+export const getPathCommands = (points: Position[]): string => {
+    if (!points || !points.length) {
+        return '';
+    }
+
+    const { x: startX, y: startY } = points[0];
+    const d = points
+        .map(({ x, y }, index, array) => {
+            const prevPoint = array[index - 1];
+            const nextPoint = array[index + 1];
+            if (!prevPoint || !nextPoint) {
+                return '';
+            }
+
+            const xc1 = (x + prevPoint.x) / 2;
+            const yc1 = (y + prevPoint.y) / 2;
+            const xc2 = (x + nextPoint.x) / 2;
+            const yc2 = (y + nextPoint.y) / 2;
+
+            return `C ${xc1} ${yc1}, ${x} ${y}, ${xc2} ${yc2}`;
+        })
+        .join(' ');
+
+    return `M ${startX} ${startY} ${d}`;
+};
+
+export const DrawingPathGroup = ({
+    isActive = false,
+    pathGroup: {
+        paths,
+        stroke: { color, size },
+    },
+    rootEl,
+}: Props): JSX.Element => {
+    let strokeWidth = size;
+    // A line has two sides and each side needs a border, so add DRAWING_BORDER_WIDTH * 2
+    let strokeWidthWithBorder = strokeWidth + DRAWING_BORDER_WIDTH * 2;
+
+    // "vector-effect: non-scaling-stroke" prevents stroke width to be scaled
+    // However, IE/Edge doesn't support it, so we have to manually inverse scale the stroke width
+    if (!isVectorEffectSupported()) {
+        if (!rootEl) {
+            return <g className="ba-DrawingPathGroup" />;
+        }
+        const { clientWidth } = rootEl;
+        const scale = clientWidth / 100;
```

Comment:
They're different. I use `clientWidth` instead of `clientHeight` because most of the documents have a smaller width compared to height. In result, the scaled stoke may be thicker in y-axis rather than thiner (low visibility).

Will add a comment about `100`.